src/app/+community-page/sub-community-list/community-page-sub-community-list.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+
+import { RemoteData } from '../../core/data/remote-data';
+import { Community } from '../../core/shared/community.model';
+
+import { fadeIn } from '../../shared/animations/fade';
+import { PaginatedList } from '../../core/data/paginated-list';
+
+@Component({
+  selector: 'ds-community-page-sub-community-list',
+  styleUrls: ['./community-page-sub-community-list.component.scss'],
+  templateUrl: './community-page-sub-community-list.component.html',
+  animations:[fadeIn]
+})
+
+export class CommunityPageSubCommunityListComponent implements OnInit {
```

Comment:
New classes/components should have TypeDocs to describe them.

Reply:
Added typedoc comment. 
I also merged latest master and fixed the errors. The branch did not contain the upgraded angular version.